apps/web-mzima-client/src/app/feed/feed.component.ts

```diff
@@ -314,6 +315,20 @@ export class FeedComponent extends MainViewComponent implements OnInit {
     });
   }
 
+  public postIsLoading(value: boolean) {
+    return (this.isLoading = value); // "There are no posts yet!" flicker is fixed for (most) places where the value here is set to true
+  }
+
+  public postsCheck() {
+    const postsHaveLoaded = this.posts.length > 0;
+    if (postsHaveLoaded) this.postIsLoading(false); // post card/content area and LoadMore button benefits from this
+    const posts = {
+      atLeastOneExists: postsHaveLoaded,
+      stillLoading: this.isLoading, // tracks this.isLoading for post card area content display
+    };
```

Comment:
If you use "this.isLoading" in the code, you don't need this stillLoading and then you can return postsHaveLoaded directly instead, to avoid complexity :)

Reply:
Ahh, yes, I understand now, it makes sense! Thanks for the explanation 🙏

Reply:
Ok, I'm yet to be able to do this differently, incase you have more suggestions.